subsys/instrumentation/timestamp/timestamp.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 Linaro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/instrumentation/instrumentation_timestamp.h>
+
+/* TODO(gromero): this is quite Arm Cortex-M-specific. Backend API? */
```

Comment:
You might want to look into include/zephyr/timing/timing.h

Reply:
Thanks!

Reply:
Done.